stdlib/http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java

```diff
@@ -270,7 +271,11 @@ static void dispatchCloseMessage(WebSocketOpenConnectionInfo connectionInfo,
         String closeReason = closeMessage.getCloseReason();
         if (onCloseResource == null) {
             if (webSocketConnection.isOpen()) {
-                webSocketConnection.finishConnectionClosure(closeCode, null);
+                if (closeCode == 1005) {
+                    webSocketConnection.finishConnectionClosure();
+                } else {
+                    webSocketConnection.finishConnectionClosure(closeCode, null);
```

Comment:
Any reason for passing null here ? should this function be a overload ?

Reply:
It is handled as an empty string in netty. They have not provided an overload which is not necessary because reason can be null.